msal4j-sdk/src/test/java/com/microsoft/aad/msal4j/TestHelper.java

```diff
@@ -188,4 +190,13 @@ static PrivateKey getPrivateKey() {
 
         return privateKey;
     }
+
+    static Map<String, String> convertJsonToMap(String jsonString) {
+        try (JsonReader reader = JsonProviders.createReader(jsonString)) {
+            reader.nextToken();
+            return reader.readMap(JsonReader::getString);
+        } catch (IOException e) {
+            throw new MsalClientException("Could not parse JSON from HttpResponse body: " + e.getMessage(), AuthenticationErrorCode.INVALID_JSON);
```

Comment:
Seems odd of the test code to throw MSAL - product exceptions.

Reply:
This was just an oversight when copying this method from `HttpResponse`, it's been removed in the latest commit and a method in `JsonHelper` is used now.